app.js

```diff
@@ -80,9 +80,12 @@ app.use(function (req, res, next) {
 // store the client secret in regardless) it makes sense
 var errors = require("./lib/errors.js").ERRORS;
 app.use(function (req, res, next) {
-    // don't authenticate OPTIONS requests for browser compatbility
+    // don't authenticate OPTIONS requests for browser compatibility
     if (req.method === "OPTIONS") return next();
 
+    // don't authenticate health checks
+    if (req.path.indexOf("/health") >= 0) return next();
```

Comment:
You're right, let's just leave with `indexOf`. Once the tests are merged into `develop`, I'll rebase from `develop` to make sure this doesn't break anything and then merge.